ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryRegistryImpl.java

```diff
@@ -56,6 +56,7 @@
 
   private final Map<QueryId, PersistentQueryMetadata> persistentQueries;
   private final Set<QueryMetadata> allLiveQueries;
+  private final Map<QueryId, QueryMetadata> allQueries;
```

Comment:
Let's see if we can get rid of allLiveQueries, since this is a bit redundant and `allQueries` an presumably be used for the same purposes.